src/apps/service/apps/jobs/submissions.clj

```diff
@@ -141,7 +141,7 @@
   [user submission]
   (let [output-dir (ft/build-result-folder-path submission)]
     (try+
-     (data-info/get-file-stats user [output-dir])
+     (data-info/get-path-info user :paths [output-dir] :filter-include "path")
```

Comment:
Technically I'm not sure this needs *anything*, because it doesn't use the return value unless it errors. I've put in `path` though just to have it do something (which is to say, only validate, because it doesn't need to fetch anything to get the path).